src/array.jl

```diff
@@ -45,6 +45,13 @@ CuArray
         # semi-hidden constructor, only called by unsafe_convert
         new{T,N}(devptr, shape)
     end
+    # Specifically convert existing CUDArt CudaArray to CuArray
```

Comment:
No, this doesn't belong here. Add it to CUDArt, which already depends on CUDAdrv.